Makefile

```diff
@@ -1,24 +1,8 @@
-k8s_version ?= 1.14.6
-globalnet ?= false
-deploytool ?= operator
+include Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e / -e clusters -e deploy)
 
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-`uname -s`-`uname -m` > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
-
-shell:
-	./.dapper -m bind -s
-
-clusters: .dapper dapper-image
```

Comment:
Not sure why you removed this, clusters needs the dapper-image rebuild in case something changed in the files, otherwise it'll pull an older image without that changes.

Reply:
But `clusters` doesn’t pull anything based on the Dapper image...

Reply:
I thought I’d replied to this already, but I can’t find my reply. `clusters.sh` doesn’t deploy anything based on the Dapper image, does it? That’s why I removed the dependency... (That, and the fact that `Makefile.inc` should be self-contained.)

Reply:
Fixed, thanks.